src/sc_generation/abis.rs

```diff
@@ -11,7 +11,7 @@ pub fn get_abis(file_path: &String) -> Vec<Vec<String>> {
             in_abi_definition = true;
```

Comment:
I feel that parsing massa-sc-runtime abis.rs would be easier
is that possible

Reply:
It may be easier indeed, but I think I'll let that be a futur refactor. It does not seem entirely trivial, and the current parsing "works". :)